stream-chat-android-client/src/main/java/io/getstream/chat/android/client/parser2/MoshiErrorLogging.kt

```diff
@@ -36,14 +34,7 @@ internal fun MoshiConverterFactory.withErrorLogging(): Converter.Factory {
         ): Converter<ResponseBody, *> {
             val originalConverter: Converter<ResponseBody, *> =
                 originalFactory.responseBodyConverter(type, annotations, retrofit)!!
-            return Converter { value ->
```

Comment:
Why are we removing this?

Reply:
I think this logger is not needed anymore.
It was due our "Moshi migration". It is complete now, so if any changes is done on the API that break our contract, it should crash, but the logger shouldn't be needed